### DISCUSSION TITLE
Original SageTV Community Forum

### DISCUSSION THREAD

**Discussion by jusjoken** [OpenSageTV/Community#1] [Announcements]
You can still access the origin SageTV forum in a now static format by visiting the link below...

[SageTV Community](https://forums.sagetv.com/)

**Comment by Sagetvmaster** [comment]
When you say 'static' you mean no input, correct?

**Reply by jusjoken** [reply]
Correct, however the static archive is not live yet so that link still goes to the non static site but Narflex would like us to stop posting there so when the archive is created it does not miss content.

Hope this helps

**Comment by wayner9** [comment]
Is there a certificate error at the old forums?  I got an error when I clicked on them by a mistake.  And I get the "Not Secure" message in the URL bar in Chrome.

**Reply by eelpout** [reply]
yes. it definitely has problems but it's still there. 

searching through it via Google can require a bit of massaging of the URL to access actual posts as the search/scripting feature on it is broken/disabled too.

i.e., you'll get this as a result from google.com for searching for "opendct forums.sagetv.com": 

`https://forums.sagetv.com/forums/showthread.php?t=62774%20`

but to access the post you'll need this: 

`https://forums.sagetv.com/forums/showthread_t_62774.html`

**Comment by eelpout** [comment]
does anyone here have any pull with Google to get the static site reindexed with .html page results instead of the php scripts which appear to be broken? 

otherwise, we have no (clean) way to search the old site for stuff without having to tinker with the results to point to the right place.

**Reply by Sagetvmaster** [reply]
I was under the impression the old site would at least be searchable.  Did I have that wrong all along?  That would suck quite a lot, actually.  Here’s what happens is a search is entered:

“404. That’s an error.

The requested URL /forums/search.php?do=process was not found on this server. That’s all we know.”

**Reply by eelpout** [reply]
yeah, figured out a while ago you have to change the Google result to use the html page and not the php, as the scripting preprocessor won't work. unfortunately, all the search results (Google) point to php. 

`https://forums.sagetv.com/forums/showthread.php?t=66577 -> https://forums.sagetv.com/forums/showthread_t_66577.html
`

it could probably be reindexed for the html pages but that can only be done by the site owner as far as I know.

the other thing that appears to be happening, at least with Bing, is since the site no longer has a valid certificate, past results I think are being removed.

**Comment by KryptoNyte-Sage** [comment]
What did it take to get immortalized in the old Sage forum with a green "online" button lit up beside your name?  You guys will live forever there!

Narflex (Jeff of course)
KeithAbbott
Jusjoken (Ken)

**Reply by KeithAbb0tt** [reply]
Yeah, the royalties are amazing...

**Comment by KryptoNyte-Sage** [comment]
You even have Internet service out there on your own private island?!  Wow.
